dandi/files/bases.py

```diff
@@ -201,13 +210,15 @@ def get_validation_errors(
             )
             return [
                 ValidationResult(
-                    origin=ValidationOrigin(
-                        name="dandi",
-                        version=dandi.__version__,
+                    origin=Origin(
```

Comment:
could be named/reused as origin_internal_dandi

Reply:
done with a defined constant